frontend/javascripts/oxalis/view/right-menu/meshes_view.js

```diff
@@ -171,6 +179,20 @@ class MeshesView extends React.Component<
       Store.dispatch(setPositionAction(seedPosition));
     };
 
+    const getComputeMeshFileButton = () => (
+      <Button
```

Comment:
I don't know whether this is already on your to do list, but this button should ideally:
- disappear (or be hidden a bit) if a mesh file was already computed (because then the button is usually not interesting to the user)
- have `loading={true}` if a compute-mesh-job is currently running (for now, you can simply call `getJobs()` and filter by `datasetName` every 60 seconds or so)

Regarding the second point: @fm3 do you think it makes sense to adapt `/api/jobs` so that it optionally accepts a datasetName parameter? Like `/api/jobs?datasetName=...`?

Reply:
Hmm, I’m not sure. Not every job is necessarily associated with a (single) dataset, so maybe we’ll have to find a more generic solution to this filtering 🤔

Reply:
Hm, I see your point. I think that a generic approach could be one of the following:
(1) allow arbitrary filtering (i.e., a get parameter for each field is supported) <-- the front-end would responsible for defining the correct filters (for the compute-mesh case, it would make sense to filter by jobType==compute_mesh + dataset=current_dataset)
(2) provide use-case-driven routes (e.g., "fetch mesh jobs for mesh ui" --> `/api/jobs/mesh?datasetName=...`; "apply merger mode" --> `api/jobs/mesh?annotationId=...`)

So, my above suggestion to allow filtering by datasetName would be a subcase of (1)

Reply:
For now I filtered the jobs by datasetName, type and state. :)